README.md

```diff
@@ -24,7 +24,7 @@ release can be found in the [compatibility matrix](docs/COMPATIBILITY.md).
 
 Semantic MediaWiki 4.0.x requires:
 
-- PHP 7.3 or above
+- PHP 7.3 up to PHP 8.0
```

Comment:
I assume this should be 7.4 due to the Composer change. Also, the heading above should be "4.1.x"

Reply:
I've just dropped the info here. Think it ended up confusing people, which would then create an issue, and then be referred to the compat matrix.